packages/@o3r/telemetry/src/environment/index.ts

```diff
@@ -146,7 +146,8 @@ export const getEnvironmentInfo = async (): Promise<EnvironmentMetricData> => {
 
   let projectName: string | undefined;
   try {
-    projectName = JSON.parse(await fs.promises.readFile(path.join(process.cwd(), 'package.json'), { encoding: 'utf8' })).name;
+    const packageJson = JSON.parse(await fs.promises.readFile(path.join(process.cwd(), 'package.json'), { encoding: 'utf8' }));
+    projectName = packageJson.config?.o3rProjectName || packageJson.name;
```

Comment:
Sounds like not the first time we have config to pass via package.json, maybe an `o3rAnalytics` or `analytics` object would be more future proof to differentiate their roles?

Maybe we should also be more explicit on the name to understand its purpose : it's not the name of the project for Otter, it is the name of the project reported in the analytics.
(Can be resolved by a subObject or a renaming)

```suggestion
    projectName = packageJson.config?.analytics?.projectName || packageJson.name;
```

Reply:
renamed into `config.o3r.telemetry`